libc/src/math/copysignf128.h

```diff
@@ -9,7 +9,7 @@
 #ifndef LLVM_LIBC_SRC_MATH_COPYSIGNF128_H
 #define LLVM_LIBC_SRC_MATH_COPYSIGNF128_H
 
-#include "src/__support/macros/properties/compiler.h"
+#include "src/__support/macros/properties/float.h"
```

Comment:
You probably want to add a cmake dep for this.

Reply:
I'll do this is a separate patch as well because `libc/src/math/CMakeLists.txt` needs to be tweaked to pass dependencies.